security/tailscale/src/opnsense/service/templates/OPNsense/Tailscale/rc.conf.d

```diff
@@ -10,6 +10,11 @@ tailscaled_enable="YES"
 tailscaled_port="{{ OPNsense.tailscale.settings.listenPort }}"
 {%    endif %}
 {%    set up_args = [] %}
+{%    if helpers.exists('OPNsense.tailscale.settings.loginTimeout') %}
+{%      do up_args.append("--timeout=" + OPNsense.tailscale.settings.loginTimeout + "s") %}
+{%    else %}
+{%      do up_args.append("--timeout=10s") %}
+{%    endif %}
```

Comment:
```suggestion
{%      do up_args.append("--timeout=" + OPNsense.tailscale.settings.loginTimeout + "s") %}
```

I'm making this a permanent setting as per migration since we have a default and require it above